steps/terraform.yaml

```diff
@@ -49,6 +49,9 @@ parameters:
   - name: kvConnectedServiceName
     default: 'azurerm-sandbox'
 
+  - name: terraformVarFile
+    default: ''
```

Comment:
why have you made this configurable?

Reply:
Hi Tim, we want to have the capability to either pass in the location of the tfvar file or use a default, which would be set

Reply:
could you provide an example?

Reply:
Repo [aks-cft-deploy](https://github.com/hmcts/aks-cft-deploy): tfvars file is in non-standard location for all environments
Repo [aks-sds-deploy](https://github.com/hmcts/aks-sds-deploy): tfvars file is in non-standard location for all environments

In both repositories, the tfvars file is in `<root_folder>/environments/<component>/<environment>.tfvars` instead of the standard path (which is `<root_folder>/environments/<environment>/<environment>.tfvars`)